app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -733,6 +733,17 @@ struct DeviceDelegateOculusVR::State {
     }
   }
 
+  void UpdateDisplayRefreshRate() {
+    if (!ovr || !IsOculusGo()) {
```

Comment:
Do we not need to set this for Quest?

Reply:
We don't need it for Quest because it is always 75hz there

Reply:
Why we still make the immersive mode to be 60hz? It seem likes in the immersive mode, we would expect the FPS be higher to kick rAFs as fast as possible.